draft-ietf-httpapi-ratelimit-headers.md

```diff
@@ -320,30 +320,34 @@ and can be sent in a trailer section.
 
 ## RateLimit-Remaining {#ratelimit-remaining-header}
 
-The `RateLimit-Remaining` response field indicates the remaining `quota-units` defined in {{request-quota}}
-associated to the client.
+The `RateLimit-Remaining` response field indicates the remaining `quota-units`
+defined in {{request-quota}} associated to the client.
```

Comment:
Not sure how I should deal with this little paragraph. It brings up the `quota-units` ABNF rule, which we're not using any more.